[Logging Service: 0]
[2024-10-31 15:51:50.493 -0300] [INFO] [cadence.virtuoso.ade.loggingservice] - HistoryRecorderManager init {"dbBasePath": "/home/cinovador/Documents/course_files/digital_logic/practice_2/logic_gates/not_gate/maestro/results/maestro"}
[2024-10-31 15:51:50.493 -0300] [INFO] [cadence.virtuoso.ade.loggingservice] - useGRPC grpc service... {"brokerAddress": "cinova05.lesc.ufc.br:37893", "serviceName": "DaVinciService_75136_1730400708", "axlSession": "fnxSession0"}
[2024-10-31 15:51:50.493 -0300] [PROCESS] [cadence.virtuoso.ade.loggingservice] - Logging Service has successfully started. Waiting for incoming messages ... {"hostname": "cinova05.lesc.ufc.br", "processid": 85128, "logfile": "/home/cinovador/Documents/course_files/digital_logic/practice_2/logs_cinovador/logs0/LoggingService0.log"}
[2024-10-31 15:51:50.757 -0300] [INFO] [cadence.virtuoso.ade.loggingservice] - logging service gRPC started {"broker address": "cinova05.lesc.ufc.br:34035"}
[2024-10-31 15:57:21.506 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "UNIQUE constraint failed: location.point, location.test, location.tool", "sql": "INSERT INTO location (tool, file, point, test, line) VALUES ('ADE GUI', '/home/cinovador/Documents/course_files/digital_logic/practice_2/logs_cinovador/logs0/virtuoso_ade_debug.log', '', '', '0');"}
[2024-10-31 15:57:21.506 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "delete from meta;"}
[2024-10-31 15:57:21.506 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "insert into meta (key, value) values('HistorySimDir','/home/cinovador/Documents/course_files/digital_logic/practice_2/logic_gates/not_gate/maestro/logic_gates/not_gate/maestro/results/maestro/ExplorerRun.0');"}
[2024-10-31 15:57:21.506 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "insert into meta (key, value) values('WorkDir','/home/cinovador/Documents/course_files/digital_logic/practice_2');"}
[2024-10-31 15:57:21.506 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "insert into meta (key, value) values('version','1');"}
[2024-10-31 15:57:21.506 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='exprOutputs.log.2.0.1' WHERE tool='Expression Evaluator' AND file='/home/cinovador/Documents/course_files/digital_logic/practice_2/logic_gates/not_gate/maestro/logic_gates/not_gate/maestro/results/maestro/ExplorerRun.0/exprOutputs.log.2.0.1';"}
[2024-10-31 15:57:21.507 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='1/logic_gates_not_gate_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cinovador/Documents/course_files/digital_logic/practice_2/logic_gates/not_gate/maestro/logic_gates/not_gate/maestro/results/maestro/ExplorerRun.0/1/logic_gates_not_gate_1/psf/spectre.out';"}
[2024-10-31 15:57:21.507 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-10-31 15:57:21.507 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-10-31 15:57:22.030 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert location failed {"reason": "attempt to write a readonly database"}
[2024-10-31 15:57:22.030 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-10-31 15:57:22.311 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-10-31 15:57:22.311 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-10-31 15:57:23.311 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-10-31 15:57:23.311 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-10-31 15:57:23.311 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-10-31 15:57:23.311 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert location failed {"reason": "attempt to write a readonly database"}
[2024-10-31 15:57:23.412 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert location failed {"reason": "attempt to write a readonly database"}
[2024-10-31 15:57:33.347 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-10-31 15:57:33.347 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-10-31 15:57:33.386 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-10-31 15:57:35.678 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert location failed {"reason": "attempt to write a readonly database"}
[2024-10-31 15:57:35.678 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-10-31 15:57:36.387 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-10-31 15:57:36.388 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-10-31 15:57:36.388 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-10-31 15:57:36.388 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-10-31 15:57:36.388 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-10-31 15:57:36.388 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-10-31 15:57:36.389 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-10-31 15:57:36.389 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-10-31 15:57:36.389 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-10-31 15:57:36.389 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-10-31 15:57:36.389 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-10-31 15:57:36.389 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-10-31 15:57:37.161 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-10-31 15:59:58.239 -0300] [INFO] [cadence.virtuoso.ade.loggingservice] - Got exit signal in processControlMessage. Will exit. {}
[2024-10-31 15:59:59.239 -0300] [PROCESS] [cadence.virtuoso.ade.loggingservice] - Current memory used: 90.8359 MBytes, peak memory used: 90.8359 MBytes {"hostname": "cinova05.lesc.ufc.br", "processid": 85128, "logfile": "/home/cinovador/Documents/course_files/digital_logic/practice_2/logs_cinovador/logs0/LoggingService0.log"}
[2024-10-31 15:59:59.240 -0300] [PROCESS] [cadence.virtuoso.ade.loggingservice] - Stopping Logging Service process. {"hostname": "cinova05.lesc.ufc.br", "processid": 85128, "logfile": "/home/cinovador/Documents/course_files/digital_logic/practice_2/logs_cinovador/logs0/LoggingService0.log"}
[2024-10-31 15:59:59.240 -0300] [PROCESS] [cadence.virtuoso.ade.loggingservice] - Logging Service stop!!!! {"hostname": "cinova05.lesc.ufc.br", "processid": 85128, "logfile": "/home/cinovador/Documents/course_files/digital_logic/practice_2/logs_cinovador/logs0/LoggingService0.log"}
